asApp/src/save_restore.c

```diff
@@ -2087,7 +2092,13 @@ STATIC void do_seq(struct chlist *plist)
 
     if (check_file(save_file) == BS_NONE) {
         printf("save_restore:do_seq - '%s' not found.  Writing a new one. [%s]\n", save_file, datetime);
-        (void)write_save_file(plist, NULL, NULL);
+        status = write_save_file(plist, NULL, NULL);
```

Comment:
`status` is not defined.

```
   ../save_restore.c:2095:9: error: use of undeclared identifier 'status'
          status = write_save_file(plist, NULL, NULL);
          ^
  ../save_restore.c:2096:13: error: use of undeclared identifier 'status'; did you mean 'stat'?
          if (status != 0) {
              ^~~~~~
              stat
```

Reply:
Sloppy of me, will fix. Good catch.

Should we have CI run also on PRs?

Reply:
https://github.com/epics-modules/autosave/pull/77